src/screens/Select.js

```diff
@@ -644,6 +645,24 @@ const SelectPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="select.clear.hover.background">
```

Comment:
this is now `theme.select.clear?.container?.hover` 

https://github.com/grommet/grommet/pull/7397

if we can change the documents to match the PR that got merged :)